src/main/java/car/tests/CarTest.java

```diff
@@ -0,0 +1,63 @@
+package car.tests;
+
+import car.model.Car;
+import car.model.CarWheel;
+import java.util.ArrayList;
+import junit.framework.Assert;
+import org.junit.jupiter.api.Test;
+
+public class CarTest {
+    ArrayList<CarWheel> wheels = new ArrayList<>();
```

Comment:
```suggestion
    List<CarWheel> wheels = new ArrayList<>();
```
Оперуй абстракцією, а не імплементацією. Таке оголошення коллекцій дуже сильно видає в тобі джуна. Май це на увазі коли будеш робити тестове